modules/ROOT/pages/update-an-api-configure-service-mesh-CLI.adoc

```diff
@@ -0,0 +1,139 @@
+
+* <<update-api-AP, Update an API using Anypoint Platform>>
++
+New versions of an API asset can be created via Exchange. Then, API instances can be updated to use this new version of the asset.
++
+Tags are updated by accesing the asset in Exchange or the API instance in API Manager, and modifying their tags through the corresponding UI.
+* <<update-api-cli,Update an API using Anypoint Service Mesh>>
++
+If the API resource exists in the cluster, it can be updated so it triggers changes in the Anypoint Platform, creating or modificating asset versions and API instances.
+
+
+[[update-api-AP]]
+== Update an API using Anypoint Platform
+
+=== Update API asset in Exchange
+To update an existing asset or create a new version of it using Anypoint Platform, perform the following steps:
+
+. Go to *Anypoint Platform > Exchange* and select the desired asset.
++
+The asset page is displayed. Now you can perform the following actions:
+. If you wish to upload a new version, click on *Add new version* on the right-side table.
+. If you wish to change the tags of a version, you can do it in the *Tags* section on the right-side panel.  
+
+=== Update API instance in API Manager
+To update an existing API instance, perform the following steps:
+
+. Go to *Anypoint Platform > API Manager*
+. Search and select your API instance in the table. 
++
+Once inside the API details page, you can perform the following actions:
+. Update the instance tags, at the *API Configuration* section.
+. Update the asset version it points to. Note that you can only update if there is a newer version of the asset, that belongs to the same product version.
+image::update-api-asset.png[Display asset update]
+
+
+[[update-api-cli]]
+== Update an API using the ASM CLI
+
+When you configured the API using the Anypoint Service Mesh autocreate feature, you can update it from the cluster as well. The update will reuse the environment information and credentials introduced in the provisioning. 
+
+To update, perform the following steps:
+
+. Using Anypoint Service Mesh, type the following command to create the API:
++
+[source,text,linenums]
+----
+asmctl api update  
+--name=_Kubernetes API resource name_ \
+--namespace=_namespace name_ \
+--version=_(Optional) new asset version for the API_ \
+--mainFile=_(Optional) name of the main file in the API specification_ \
+--specPath=_(Optional) path to the API specification file_ \
+--specTags=_list of comma separated tags to be added to the asset on Exchange_ \
+--instanceTags=_list of comma separated tags to be added to the instance on API Manager_ 
+----
++
+[%header%autowidth.spread]
+|===
+| Parameter Name | Description | Required or Optional | Default Value
+| `name` | The name of the Kubernetes API resource that you want to provision, for example, `orders-api`. | Required | Not applicable
+| `namespace` | The namespace in which you want to create the adapter, for example, `mulesoft-apps`. | Optional | default
```

Comment:
Can you check if the namespace remains optional? I remember Rodri had removed the optional part for many of the commands.

Reply:
Yes, this is what we were discussing the other day. This whole table must be deleted though, the correct table is at the bottom of the page.